### DISCUSSION TITLE
Run a job once

### DISCUSSION THREAD

**Discussion by jpastorm** [go-co-op/gocron#256] [Q&A] [answered]
Greetings,

I am trying to run the jobs only once, and I would like to know if there is something for this or if I must necessarily run **every** then if I don't put it, it gives me an error

```
s.Every(1).Tuesday().At(t).Tag(UUID).Do(job.Fn, jobFnParams...)
s.RemoveByTag(UUID)
```
I am trying this

```
s.Tuesday().At(t).Tag(UUID).Do(job.Fn, jobFnParams...)
```
Now I'm doing this, I don't know if I'm making a mistake and something similar already exists.

**Comment by JohnRoesler** [comment, marked as answer]
Take a look at https://pkg.go.dev/github.com/go-co-op/gocron#Scheduler.LimitRunsTo

**Reply by jpastorm** [reply]
Thanks!
